standalone/src/main/java/io/pravega/local/InProcPravegaCluster.java

```diff
@@ -487,6 +487,10 @@ public void close() throws Exception {
             }
         }
 
+        if (secureZK) {
+            ZKTLSUtils.unsetSecureZKClientProperties();
```

Comment:
Could the fact of running these tests in parallel now be problematic when calling this static method concurrently?

Reply:
We have tried it multiple times and test cases are passing consistently.